lib/tls.js

```diff
@@ -232,4 +232,6 @@ exports.TLSSocket = require('_tls_wrap').TLSSocket;
 exports.Server = require('_tls_wrap').Server;
 exports.createServer = require('_tls_wrap').createServer;
 exports.connect = require('_tls_wrap').connect;
-exports.createSecurePair = require('_tls_legacy').createSecurePair;
+exports.createSecurePair = internalUtil.deprecate(function() {
+  require('_tls_legacy').createSecurePair.apply(null, arguments);
+}, 'tls.createSecurePair is deprecated. Use tls.TLSSocket instead');
```

Comment:
Very minor nit:
Looks like majority of existing deprecation warnings in core lib have period `.` after the last word `instead`